build.config.ts

```diff
@@ -0,0 +1,19 @@
+import { fileURLToPath } from "node:url";
+import { defineBuildConfig } from "unbuild";
+
+export default defineBuildConfig({
+  entries: [
+    "src/index.ts",
```

Comment:
You don't need the extension:
```suggestion
    "src/index",
```